client/domain/TaskObjectFactory.js

```diff
@@ -67,6 +69,19 @@ tie.factory('TaskObjectFactory', [
       return this._outputFunctionName;
     };
 
+    Task.prototype.getOutputFunctionNameWithoutClass = function() {
+      if (this._outputFunctionName.startsWith(
```

Comment:
This will fail in IE. Use indexOf(...) === 0 instead.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith